src/views/List.jsx

```diff
@@ -22,7 +26,22 @@ export function List({ data }) {
 		setFilterInput('');
 	};
 
-	return (
+	const handleAddItem = () => {
+		/* navigate to add item view */
+		navigate('/add-item');
+	};
+
+	return data.length == 1 ? (
```

Comment:
Would this work for `my test list` where there isn't an extra empty document in the collection? That may be a unique enough scenario that it may not matter here but just curious if there would be cases where there is one item that isn't the empty one.

Reply:
Good point! I'm not sure if this is something to be updated in how we make collections in `firebase.js`, but we fixed it in `List.jsx` so that whether or not an extra empty document is created in a new list collection, our logic should work. 

We updated the code so that it filters the values of a collection shopping list and then checks if it contains items that has the name property.